json-rpc/src/runtime.rs

```diff
@@ -39,11 +68,11 @@ pub fn bootstrap(
     chain_id: ChainId,
 ) -> Runtime {
     let runtime = Builder::new()
-        .thread_name("rpc-")
+        .thread_name("json-rpc")
```

Comment:
So, that's why all the logs had thread names of `<component>-`

Reply:
yes, after checked log, I think remove last "-" looks better, what you think?